static/app/utils/issueTypeConfig/errorConfig.tsx

```diff
@@ -106,6 +111,21 @@ const errorHelpTypeResourceMap: Record<
       linksByPlatform: {},
     },
   },
+  [ErrorHelpType.DYNAMIC_SERVER_USAGE]: {
+    resources: {
+      description: tct(
+        '[errorTypes] occur in Next.js applications when a route is statically generated, but uses dynamic server values at runtime. To learn more about how to fix these errors, check out these resources:',
+        {errorTypes: <b>Dynamic server usage errors</b>}
+      ),
+      links: [
+        {
+          text: t('Resolving "app/ Static to Dynamic Error" in Next.js'),
+          link: 'https://nextjs.org/docs/messages/app-static-to-dynamic-error',
+        },
+      ],
+      linksByPlatform: {},
```

Comment:
@roggenkemper can we just omit that if we don't need it?

Reply:
right now it is required, i can take a look later at making optional